internal/pips/pips.go

```diff
@@ -42,20 +41,18 @@ func (p PIP) Name() string {
 }
 
 func (p PIP) ConvertSIPPath(path string) string {
-	switch {
-	case filepath.Base(path) == "Prozess_Digitalisierung_PREMIS.xml":
-		parent := filepath.Base(filepath.Dir(path))
-		return filepath.Join(
-			"metadata",
-			fmt.Sprintf("Prozess_Digitalisierung_PREMIS_%s.xml", parent),
-		)
-	case filepath.Base(path) == "metadata.xml":
-		return filepath.Join("objects", p.Name(), "header", "metadata.xml")
-	case filepath.Base(path) == "UpdatedAreldaMetadata.xml":
-		return filepath.Join("metadata", "UpdatedAreldaMetadata.xml")
-	case strings.HasPrefix(path, "content"):
+	switch name := filepath.Base(path); name {
+	case "Prozess_Digitalisierung_PREMIS.xml":
+		return filepath.Join("metadata", name)
+	case "metadata.xml":
+		return filepath.Join("objects", p.Name(), "header", name)
+	case "UpdatedAreldaMetadata.xml":
+		return filepath.Join("metadata", name)
```

Comment:
`Prozess_Digitalisierung_PREMIS.xml` and `UpdatedAreldaMetadata.xml` could now go in the same case.